spec/doorkeeper-jwt/config_spec.rb

```diff
@@ -35,6 +35,21 @@
     end
   end
 
+  describe 'use_application_secret' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.